pkg/storage/mysql/mysqlstore_test.go

```diff
@@ -0,0 +1,296 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package mysql
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+
+	"github.com/stretchr/testify/require"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+const (
+	sqlStoreDBURL = "root:my-secret-pw@tcp(127.0.0.1:3306)/"
+)
+
+// For these unit tests to run, you must ensure you have a SQLDB instance running at the URL specified in sqlStoreDBURL.
```

Comment:
Please have MySQL spooled up by the tests in this package (`TestMain`) because they're the only unit tests that require it.

Reply:
Spooling inside the test file now